Setup/Product.wxs

```diff
@@ -135,6 +138,312 @@
         <File Source="..\GitExtensions\bin\$(var.Configuration)\System.Runtime.InteropServices.RuntimeInformation.dll" />
       </Component>
 
+            <Component Id="Microsoft.Win32.Primitives.dll" Guid="*">
```

Comment:
💡 Would be good to switch the new lines to use a 2-space indent like the rest of the file.

Reply:
Again. I do this on purpose.